### ISSUE TITLE
'Add' Primary Button lacks focus indicator

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#3623]
## Environment
Themes

> Operating system
Windows 7

> Browser
Chrome and Firefox

## Detailed description
The + button has no focus indictor, instead displaying "Add" on focus as a tooltip
it's an extremely unusual way of indicating focus. As well, it is the same indicator as the hover. 
Note that when the item is selected, there is a (faint) inner focus rectangle around the + symbol on hover. Unclear why it is not visible when not hovered. Note also that Secondary buttons have a (poor) focus rectangle on the + button.

> What did you expect to happen?
I expected it to receive a standard dotted rectangle focus indicator. 

> What happened instead?
There is no focus indicator. For a user who is in a magnified view, they may not even see the "Add" pop up content, so it is a very poor indicator that the button has focus.

> What WCAG 2.1 checkpoint does the issue violate?
[Focus Visible](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html)

## Steps to reproduce the issue

1. Open up the themes page and select Button from the left nav
2. Click or otherwise put focus on the first button, then tab to the "+" button
3. To see the faint focus indicator that exists, select the Add button, then hover over it.

**Comment by tw15egan** [comment]
This was fixed in https://github.com/carbon-design-system/carbon/pull/3598

Example: https://deploy-preview-3598--the-carbon-components.netlify.com/component/button--primary.html
